Firestore/core/src/firebase/firestore/model/snapshot_version.h

```diff
@@ -38,6 +43,27 @@ class SnapshotVersion {
   /** Creates a new version that is smaller than all other versions. */
   static const SnapshotVersion& None();
 
+#if defined(__OBJC__)
+  SnapshotVersion(FSTSnapshotVersion* version) {  // NOLINT(runtime/explicit)
+    if (![version isEqual:[FSTSnapshotVersion noVersion]]) {
```

Comment:
Is this check even required? Why not just directly initialize the timestamp?

```c++
SnapshotVersion(FSTSnapshotVersion* version)
    : timestamp_{version.timestamp} {
}
```
Or if Timestamp doesn't define a conversion to FIRTimestamp:
```c++
SnapshotVersion(FSTSnapshotVersion* version)
    : timestamp_{version.timestamp.seconds, version.timestamp.nanoseconds} {
}
```

Reply:
Done. not really required as far as the logic of `noVersion` are the same between objc and C++, which seems the case as both are of epoch 0.